Website/DesktopModules/Hotcakes/Core/Controllers/CheckoutController.PaymentError.cs

```diff
@@ -44,6 +45,17 @@ public partial class CheckoutController
 		public ActionResult PaymentError()
 		{
 			CheckoutViewModel model = PaymentErrorSetup();
+            if (HccApp.CurrentStore.Settings.PaymentCreditCardGateway == "15011DF5-13DA-42BE-9DFF-31C71ED64D4A")
```

Comment:
Is there not an enum to use for this comparison? If not, then we should use a constant to compare so the heap doesn't have a series of new strings added to memory for a simple and frequently repeated IF statement.